R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
Yeah I guess you are right this function should

1. If GITHUB_PAT is set, use that (say `Using github PAT from envvar GITHUB_PAT`)
2. If GITHUB_PAT is not set and in a CI, use the default PAT (with appropriate message?)
3. Otherwise return NULL